create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/publication/exception/FileNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.download.publication.file.publication.exception;
+
+import nva.commons.exceptions.ApiGatewayException;
+import org.apache.http.HttpStatus;
+
+public class FileNotFoundException extends ApiGatewayException {
+
+    public FileNotFoundException(String message) {
+        super(message);
+    }
+
+    @Override
+    protected Integer statusCode() {
+        return HttpStatus.SC_BAD_REQUEST;
```

Comment:
Why is this a bad request?

Reply:
Publication is found, but provided file identifier is bogus. Would 404 be more appropriate?